src/js/components/Accordion/stories/CustomThemed/Custom.js

```diff
@@ -29,14 +29,7 @@ const customAccordionTheme = {
       color: 'hotpink',
     },
     border: undefined,
-    panel: {
-      // border: {
-      //   side: 'horizontal',
-      //   size: 'medium',
-      //   color: '#DADADA',
-      //   style: 'dotted',
-      // },
-    },
+    panel: {},
```

Comment:
If this is an empty object, should we just remove it?

Reply:
Removed